src/main/java/io/aiven/kafka/connect/opensearch/BulkProcessor.java

```diff
@@ -374,20 +374,22 @@ private BulkResponse execute() throws Exception {
                         return response;
                     }
                     for (final var itemResponse : response.getItems()) {
-                        if (!itemResponse.getFailure().isAborted()) {
-                            if (responseContainsMalformedDocError(itemResponse)) {
-                                handleMalformedDoc(itemResponse);
-                            } else if (responseContainsVersionConflict(itemResponse)) {
-                                handleVersionConflict(itemResponse);
-                            } else {
-                                throw new RuntimeException(
-                                        "One of the item in the bulk response failed. Reason: "
+                        if(itemResponse.isFailed()) {
```

Comment:
```suggestion
                        if (itemResponse.isFailed()) {
```
The checkstyle failed:

```
Error: eckstyle] [ERROR] /home/runner/work/opensearch-connector-for-apache-kafka/opensearch-connector-for-apache-kafka/src/main/java/io/aiven/kafka/connect/opensearch/BulkProcessor.java:377:25: 'if' is not followed by whitespace. [WhitespaceAfter]
```